multiraft/multiraft.go

```diff
@@ -720,6 +722,7 @@ func (s *state) handleWriteResponse(response *writeResponse, readyGroups map[uin
 					NodeID:     NodeID(cc.NodeID),
 					ChangeType: cc.Type,
 					ConfState:  *cs,
+					Payload:    payload,
```

Comment:
What I'm concerned about here is that the conf change is applied before we know whether or not the transaction will commit. What if the transaction fails to commit?